Integrations/FireEyeHelix/FireEyeHelix.py

```diff
@@ -0,0 +1,1643 @@
+                # move DocCount to tail
+                group_by_keys.remove('DocCount')
+                group_by_keys.append('DocCount')
+                hr += tableToMarkdown('Group By', context['GroupBy'], headers=group_by_keys)
+
+        return hr, {f'{INTEGRATION_CONTEXT_NAME}Search({dt_query})': context}, raw_response
+    else:
+        # API should not return an empty result matching this case, this is a fail safe
+        return f'{INTEGRATION_NAME} - Search did not find any result.', {}, {}
+
+
+def build_single_list_result(raw_response):
+    """Builds a list result from API response
+
+    Args:
+        raw_response: API response to alert call
+
+    Returns:
+        List result
+    """
+    list_id = raw_response.get('id')
+    title = f'{INTEGRATION_NAME} - List {list_id}:'
+    context_entry = build_transformed_dict(raw_response, LISTS_TRANS)
+    context = {
+        f'{INTEGRATION_CONTEXT_NAME}.List(val.ID && val.ID === obj.ID)': context_entry
+    }
+    # Creating human readable for War room
+    human_readable = tableToMarkdown(title, context_entry, headerTransform=pascalToSpace)
+    # Return data to Demisto
+    return human_readable, context, raw_response
+
+
+''' COMMANDS '''
+
+
+def test_module(client: Client, *_) -> Tuple[str, Dict, Dict]:
+    """Performs a basic GET request to check if the API is reachable and authentication is successful.
+
+    Args:
+        client: Client object with request
+        args: Usually demisto.args()
+
+    Returns:
+        'ok' if test successful.
+
+    Raises:
+        DemistoException: If test failed.
+    """
+    client.test_module()
+    return 'ok', {}, {}
+
+
+def fetch_incidents(client: Client, fetch_time: str, last_run: Dict) -> Tuple[List, Dict]:
+    """Uses to fetch incidents into Demisto
+    Documentation: https://github.com/demisto/content/tree/master/docs/fetching_incidents
+
+    Args:
+        client: Client object with request
+        fetch_time: From when to fetch if first time, e.g. `3 days`
+        last_run: Last fetch object.
+
+    Returns:
+        incidents, new last_run
+    """
+    timestamp_format = '%Y-%m-%dT%H:%M:%S.%fZ'
+    # Get incidents from API
+    if not last_run:  # if first time running
+        new_last_run = {'time': parse_date_range(fetch_time, date_format=timestamp_format)[0]}
+    else:
+        new_last_run = last_run
+    incidents: List = list()
+    raw_response = client.list_alerts(created_at__gte=new_last_run.get('time'))
+    alerts = raw_response.get('results')
+    if alerts:
+        last_incident_id = last_run.get('id', 0)
+        # Creates incident entry
+        incidents = [{
+            'name': f"{INTEGRATION_NAME}: {alert.get('id')}",
+            'occurred': alert.get('created_at'),
+            'severity': alert_severity_to_dbot_score(alert.get('severity')),
+            'rawJSON': json.dumps(alert)
+        } for alert in alerts if alert.get('id') > last_incident_id]
```

Comment:
if we can filter alerts by time, why should we manage IDs? maybe we can discard this logic?

Reply:
Because the the filter is for Greater or Equal, so we might get several alerts that have the same time as the alert that was filtered  - so we need to further filter them by ID.